src/main/java/it/pn/frontend/e2e/pages/destinatario/personaFisica/DeleghePage.java

```diff
@@ -74,9 +74,9 @@ public boolean CercaEsistenzaDelega(String nome,String cognome) {
     public void clickRevocaButton(String nome, String cognome) {
 
         By menuButton = By.xpath("//div[@data-testid='delegates-wrapper']//td[@role='cell' and div/p[contains(text(),'"+nome+" "+cognome+"')]]/following-sibling::td[@role='cell']//button[@data-testid='delegationMenuIcon']");
```

Comment:
verifca se inserito già in documento id

Reply:
non mi serve id,  a meno che tu riesca a mettere qualcosa che mi indica il nome e cognome della delega